mobile-widgets/qml/MapWidget.qml

```diff
@@ -69,17 +69,10 @@ Item {
 		ParallelAnimation {
 			id: mapAnimation
 			CoordinateAnimation {
-				target: map
-				property: "center"
-				to: map.newCenter
-				duration: 2000
+				target: map; property: "center"; to: map.newCenter; duration: 2000;
```

Comment:
we are not 100% consistent with that, but in general we went with one property per line unless there is a reason not to...